gradle.properties

```diff
@@ -11,4 +11,4 @@ org.gradle.caching = true
 includesRegex = .*
 
 # Require spotless to fetch license header dates from Git history
-spotlessSetLicenseHeaderYearsFromGitHistory = false
\ No newline at end of file
+spotlessSetLicenseHeaderYearsFromGitHistory = false
```

Comment:
In the history of this PR, services version # went from 33 to 34 then back to 33. Was the going back to 33 a mistake? It looks to me like it got accidentally merged in when a different change was reverted.
`version = 0.33.0-SNAPSHOT`

Reply:
Hmm yes seems like an accident. 🤔